security/pkg/registry/kube/service.go

```diff
@@ -31,10 +31,10 @@ import (
 )
 
 // ServiceController monitors the service definition changes in a namespace. If a
-// new service is added with "alpha.istio.io/kubernetes-serviceaccounts" annotation
-// enabled, the corresponding service account will be added to the identity registry
+// new service is added with "alpha.istio.io/kubernetes-serviceaccounts" and
```

Comment:
s/and/or, the two service accounts are logical 'or', i.e., customers should only specify one of them.

Reply:
Done.